### DISCUSSION TITLE
New Lesson Content section: FAQ or Q&A

### DISCUSSION THREAD

**Discussion by AngeloThys** [TheOdinProject/theodinproject#4391] [Ideas]
A **Frequently Asked Questions** (FAQ) or **Questions & Answers** (Q&A) section on each lesson.

I sometimes find myself posing a question in discord after consulting due resources -_MDN, StackOverflow, internet searches and Discord server searches_- but answers seem hard to find and therefor lost, be it in the form of thinly spread messages or a thread, whilst likely having been asked before.

There is already a [recommendation about making questions into normalized threads](https://discordapp.com/channels/505093832157691914/1175205552708653076), which appears to be a simpler, more manageable solution to this issue. However, the two don't seem incompatible: each _x_ amount of time, the most reasonable threads could be transformed into _documentation_ for the **Q&A** section of the relevant lesson.

**Comment by KevinMulhern** [comment]
Sorry, I thought I replied to this the other day. Thanks for the suggestion @AngeloThys 💪   

I think I have a pretty good idea of what you're pitching here, but just to make sure we're on the same page. Would you mind dropping in some examples of the kind of questions and answers that would be on these faqs? The stuff you've seen on Discord would be good real world examples to see.

**Comment by gitclonehama** [comment]
What do you think about simply adding a comment/ discussions section to each lesson (think like a YouTube comment section people could click to expand and see discussion with up/downvoting system). I can create my own idea post, but it is similar in essence to this one and I wanted to see your thoughts. 

Although we have the discord community, this would keep the focus to each lesson as well as help create more of that community feel right here on the website they are learning from. This would additionally allow comments to more easily enable community additions to lessons (whether that is clarifying a part or suggesting changes without having to know or go through all the details of navigating GitHub).

**Comment by codeandflu** [comment]
I had a similar suggestion too, I saw the documentation resource maintained by free Code Camp, which can be availed offline too. As I find myself studying without the internet a lot. Not sure about others. 

Can we have an option to download lessons and all assosciated links within it.

and new updates would be refreshed when we visit the page again


Wouldn't this be a good addition?
